src/api/object.js

```diff
@@ -5,6 +5,7 @@ const DAGLink = require('ipfs-merkle-dag').DAGLink
 const promisify = require('promisify-es6')
 const bs58 = require('bs58')
 const bl = require('bl')
+const cleanMultihash = require('../clean-multihash')
 
 module.exports = (send) => {
   const api = {
```

Comment:
You can make that even nicer, 

```js
const opts = Object.assign({enc: 'base58'}, options)
```

and it will override `enc` if it was given in `options`